packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -179,7 +179,10 @@ export const PaymentContextProvider = ({
     );
     return (
       productPrices?.data?.details?.lineItems?.map((item) => {
-        const duration = planTypes[item.price.id] as PlusPriceType;
```

Comment:
Sometime `planTypes[item.price.id]` could not be found due to some cache/GB update during onboarding. 
This is a quick fix moving logic into paddle billingCycle.